projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.js

```diff
@@ -16,6 +16,7 @@ import { store as noticesStore } from '@wordpress/notices';
 /**
  * External dependencies
  */
+import React from 'react';
```

Comment:
I'm wondering if we need to import `React` here, I tested locally removing it and couldn't notice a difference.